src/overlays/actors/ovl_Obj_Um/z_obj_um.c

```diff
@@ -46,131 +73,1949 @@ const ActorInit Obj_Um_InitVars = {
+                            potFragments[(s32)(Rand_ZeroOne() * ARRAY_COUNT(potFragments))]);
+    }
+}
+
+void ObjUm_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    ObjUm* this = THIS;
+    GraphicsContext* gfxCtx = play->state.gfxCtx;
+    Mtx* mtx_s3;
+    Gfx* spFC[] = {
+        NULL, gUmBrokenMinigamePotDL, gUmMinigamePotDL, gUmMinigamePotDL, gUmMinigamePotDL, object_um_DL_0067C0
+    };
+    Gfx* spE4[] = { NULL, NULL, object_um_DL_004B60, object_um_DL_0043E0, NULL, NULL };
+    f32 spCC[] = { 0.0f, 1070.0f, 1070.0f, 1070.0f, 1070.0f, 2100.0f };
+    Vec3f spC0 = gZeroVec3f;
+    Vec3f calcPotPos;
+    f32 spB0;
+
+    spC0.y += 1700.0f;
+
+    if (limbIndex == UM_LIMB_WAGON_CART_BED) {
+        Vec3f spA4 = { 2000.0f, 1070.0f, 0.0f };
+
+        Matrix_MultVec3f(&spA4, &this->unk_308);
+        this->flags |= OBJ_UM_FLAG_DRAWN_FLOOR;
+    }
+
+    if (limbIndex == UM_LIMB_WAGON_CART_BED) {
+        Vec3f sp98 = { 2500.0f, 200.0f, 0.0f };
+
+        Matrix_MultVec3f(&sp98, &this->cartBedPos);
+    }
+
+    if (limbIndex == UM_LIMB_WAGON_CART_BED) {
+        Vec3f* potPos;
+        Vec3f sp88;
+        Vec3s sp80;
+        s32 i;
+        f32 sp70[] = { 2000.0f, 0.0f, -2000.0f };
+
+        //! FAKE
+        if (!i) {}
+
+        sp80.x = 0;
+        sp80.z = 0;
+        sp88.x = 6800.0f;
+        OPEN_DISPS(gfxCtx);
+
+        for (i = 0; i < ARRAY_COUNT(this->potsLife); i++) {
+            sp88.z = sp70[i];
+            sp88.y = spCC[this->potsLife[i]];
+
+            if (this->potsLife[i] == 5) {
+                sp80.y = 0x4000;
+            } else {
+                sp80.y = -0x4000;
+            }
+
+            Matrix_Push();
+            Matrix_TranslateRotateZYX(&sp88, &sp80);
+            mtx_s3 = Matrix_NewMtx(gfxCtx);
+            potPos = &this->potPos[i];
+            Matrix_MultVec3f(&spC0, &calcPotPos);
+            SkinMatrix_Vec3fMtxFMultXYZW(&play->viewProjectionMtxF, &calcPotPos, potPos, &spB0);
+
+            if (this->wasPotHit[i]) {
+                this->wasPotHit[i] = false;
+                if (this->potsLife[i] == 1) {
+                    ObjUm_SpawnFragments(play, &calcPotPos);
+                } else {
+                    EffectSsHitmark_SpawnFixedScale(play, 0, &calcPotPos);
+                }
+            }
+            Matrix_Pop();
+
+            if (mtx_s3 != NULL) {
+                if (play) {}
+                gSPMatrix(POLY_OPA_DISP++, mtx_s3, G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+
+                if (spFC[this->potsLife[i]] != NULL) {
+                    s32 pad;
+
+                    gSPDisplayList(POLY_OPA_DISP++, spFC[this->potsLife[i]]);
+
+                    if (spE4[this->potsLife[i]] != NULL) {
+                        gSPDisplayList(POLY_OPA_DISP++, spE4[this->potsLife[i]]);
+                    }
+                }
+            } else {
+                // !@bug: skips CLOSE_DISPS
```

Comment:
```suggestion
                //! @bug: skips CLOSE_DISPS
```
and I thought we didn't use colons, but there's a lot with.

Reply:
28 with colon and 33 without... What should we do?

Reply:
Doxygen style seems to be do not use colons.